src/plugins/intel_cpu/src/emitters/x64/jit_emitter.hpp

```diff
@@ -3,19 +3,29 @@
 //
 
 #pragma once
-
 #include <ie_common.h>
 #include <cpu/x64/jit_generator.hpp>
 
 #include "snippets/snippets_isa.hpp"
 #include "snippets/generator.hpp"
 #include <node.h>
 
+#include "openvino/runtime/threading/thread_local.hpp"
+
 #include <set>
 
+#ifndef _WIN32
+#include <cxxabi.h>
+#endif
+
+using namespace ov::threading;
+
 namespace ov {
 namespace intel_cpu {
 
+class jit_emitter;
+extern std::shared_ptr<ThreadLocal<jit_emitter*>> g_snippets_err_handler;
```

Comment:
The same suggestion as for class field: can we give a more common name for this handler? E.g. `g_specific_segfault_handler`

Reply:
done, thanks!